README.md

```diff
@@ -188,6 +188,39 @@ tableOf("a", "b", "result")
 
 Up to 4 columns are supported.
 
+### Asserting against Java's `Optional` type
```

Comment:
Think adding this to the README is overkill, we don't include every assertion here as it would make things quite long. I think it being included in the javadoc is sufficient.

Reply:
will remove it from the PR tonight after work. Thanx for the feedback